src/components/MVP/Header/Header.tsx

```diff
@@ -40,15 +40,13 @@ const Header = () => {
     },
     { path: '/about-us', label: 'About us', className: styles.aboutLink },
   ].map((i) => (
-    <Link href={i.path} key={i.path}>
-      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-      <a
-        className={classnames('usa-nav__link', i.className, {
-          'usa-current': i.path === router.pathname,
-        })}>
-        <span>{i.label}</span>
-      </a>
-    </Link>
+    <NavLink
+      href={i.path}
+      key={i.path}
+      className={classnames('usa-nav__link', i.className)}
+      exact={i.path === '/'}>
+      <span>{i.label}</span>
+    </NavLink>
```

Comment:
It looks like the existing mvp doesn't assign an active class to the Home link when clicked. I'm fine with keeping it for consistency, although it does disappear once one of the 'On this page' links are clicked.

Reply:
@abbyoung Ohh good catch, I didn't even notice! @noahfirth how do you feel about this?

Reply:
Let's not make an exception, I kind of like how it looks with the active state marker next to it on the home page.